frontend/pages/SoftwarePage/SoftwareVulnerabilityDetailsPage/SoftwareVulnSoftwareVersions/SwVulnSwTableConfig.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { Column } from "react-table";
+
+import { IVulnerabilitySoftware } from "interfaces/software";
+
+import PATHS from "router/paths";
+
+import LinkCell from "components/TableContainer/DataTable/LinkCell";
+import TextCell from "components/TableContainer/DataTable/TextCell";
+import ViewAllHostsLink from "components/ViewAllHostsLink";
+import SoftwareIcon from "pages/SoftwarePage/components/icons/SoftwareIcon";
+
+interface ICellProps {
+  cell: {
+    value: number | string | IVulnerabilitySoftware;
+  };
+  row: {
+    original: IVulnerabilitySoftware;
+  };
+}
+
+interface IStringCellProps extends ICellProps {
+  cell: {
+    value: string;
+  };
+}
+
+interface INumberCellProps extends ICellProps {
+  cell: {
+    value: number;
+  };
+}
+
+interface IVulnSoftwareCellProps extends ICellProps {
+  cell: {
+    value: IVulnerabilitySoftware;
+  };
+}
+
+const generateColumnConfigs = (isPremiumTier: boolean): Column[] => {
+  const configs = [
+    {
+      Header: "Name",
+      disableSortBy: true,
+      accessor: "name",
+      Cell: ({ row }: ICellProps) => {
+        const { name, id } = row.original;
+        return (
+          <LinkCell
+            path={PATHS.SOFTWARE_VERSION_DETAILS(id.toString())}
+            value={
+              <>
+                <SoftwareIcon name={name} />
+                <span className="vuln-sw-name">{name}</span>
+              </>
+            }
+          />
+        );
+      },
+    },
+    {
+      Header: "Version",
+      disableSortBy: true,
+      accessor: "version",
+      Cell: ({ cell }: IStringCellProps) => <TextCell value={cell.value} />,
+    },
+    {
+      Header: () => (
+        <>
+          Resolved in <div className="resolved-suffix">version</div>
+        </>
+      ),
+      disableSortBy: true,
+      accessor: "resolved_in_version",
+      Cell: ({ cell }: IStringCellProps) => <TextCell value={cell.value} />,
+    },
+    {
+      Header: "Hosts",
+      disableSortBy: true,
```

Comment:
Sortable or not?

Reply:
@rachaelshaw

Reply:
https://www.figma.com/file/iWRGEtryWkdPru8E9sJRiK/%2315919-Add-Vulnerabilities-page?node-id=352%3A11767&mode=dev

The responsive shows sort, but the original says default sort but no sort button?

Reply:
This should be default sorted on the frontend?

Reply:
This should be default sorted on the frontend?

Reply:
Default sorted, but user can't change the sort. That feature got cut during design review and looks like I missed removing it on the responsive, will get rid of it in Figma